src/index.js

```diff
@@ -0,0 +1,6 @@
+import CKEditor from './ckeditor.jsx';
```

Comment:
We need a test for the file since we're changing the output (`dist/ckeditor.js` file). It will not export the default component anymore.

The test should check whether `dist/index.js` exports an object and whether the object has two properties with proper types.

Reply:
Also, the file misses the license banner.